app/models/concerns/project_associations.rb

```diff
@@ -41,6 +41,7 @@ module ProjectAssociations
     has_many :commit_flags, -> { order(time: :desc) }, through: :analysis_sloc_sets
     has_one :project_vulnerability_report
     has_many :commit_contributors
+    has_many :project_sboms # this might get changed
```

Comment:
Remove the comment.

Reply:
Change this to
```
has_many :project_sboms, as: :sboms
```

Use `project.sboms` instead of `project.project_sboms` elsewhere.